playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.jsx

```diff
@@ -27,6 +27,7 @@ type DatePickerProps = {
   hideIcon?: Boolean,
   hideLabel?: Boolean,
   id?: String,
+  inLine?: Boolean,
```

Comment:
Everywhere else we've passed in this prop it's been fully downcased. Should we standardise this?